The Lizard's Gas (Station) Update

san7890 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

> From the depths of outer space, something floats down into our horizon. 

![image](https://user-images.githubusercontent.com/34697715/150719920-5bc8bbe1-a095-4239-97b0-02c5a8b49a41.png)

A gas station! Specifically, one themed around lizards! However, you should probably note that no lizards were involved in the creation of this gas station (hence why it's a crude mimickry), although it did source certain goods from local populations. The grand majority of it is food that can be seen across this small pocket of the universe, though. Don't worry, that blank roadsign will not show up in game (just a placeholder so you don't get spoiled on how epic this space ruin is).

If you look closely, the center crowning jewel of it all? A (little bit over half) maximally filled plasma tank! You can definitely fill up your shuttle and fly off with plasma as your fuel! You use plasma for fuel, right? Right? 

A small pocket can be found with gear to help maintain the plasma tank and the rest of the station adjacent, a small seating area (set up your own chairs!) in the top right, and a billboard in the bottom right exclaiming exactly where you are. The Lizard's Gas! It... might have gotten a bit graffitied over the years, though.

Here's an area overlay, just in case I screwed those up:

![image](https://user-images.githubusercontent.com/34697715/150719472-9fd10a3d-1cef-4b9b-9c08-1624185aa931.png)

It's meant to mimick those old, crummy gas stations in the middle of nowhere where all the ads are on standard printer paper, the restroom is non-existent, and it's scuffed in every aspect except cleanliness (it's clean because it hasn't been touched for fifty years or more oooo spooky). However, whenever I go on road trips, I wanted to give the same sense of finding a few gems if you look close enough in this as well.


<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

![image](https://user-images.githubusercontent.com/34697715/150628286-f07c69a7-edb5-4147-857b-3066b089e453.png)

That image above you is gasthelizards.dmm, and it's been mediocre-to-bad since it was added 2016-ish (or earlier).

It's commonly held to be one of the blights of the mapping scene here in our repo. When I spoke to people about it, I think I heard "make it more about gas or make it more about lizards". This does both. If you're worried about the phrase "GAS THE LIZARDS!" being lost to time, or feel saddened by the fact that utter dogshit is departing this repo, I've paid plenty of homage to how shitty the old map was in it's design and spriting.

I love the idea of random spawners, and this map leverages them to their full capacity in every potential crevice (especially by adding some more as well as containers for those random spawners). I hope that the _chance_ to get epically rare swag gamer-epic loot (which is epic) gives it a bit more longetivity than if I just threw in a mix of overpowered stuff guaranteed to anyone who should happen upon this ruin. There's also some extra spriting done, and you could code-dive to see it... but wouldn't it be more fun to seek it out and press your luck?

Achieves the challenge thrown down by #62195.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and it's effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
add: Introducing: The Lizard's Gas! A brand new (gas) station for you to go out and refuel your ships and your stomachs. It's been a while since the power's been turned on, but everything is just the way it was left all those years ago. Explore!
del: gasthelizards.dmm has been deleted. you will never see gasthelizards.dmm ever again.
config: since gasthelizards.dmm is removed, the config has been updated as well. make sure it's up-to-date now that we've gotten rid of the dang ol' thing and replaced it with new hotness.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

_As always, any and all feedback is appreciated._

san7890 committed:
first change

i'm hopping between branches and i dont wanna lose muh changes

san7890 committed:
part two

needs more work but i like it

san7890 committed:
billboarding

soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul. soul.

san7890 committed:
FUCK

mumble mumble

san7890 committed:
you'd think i know how lists work

you'd think

san7890 committed:
Update gasthelizards.dmm

san7890 committed:
swap branches moment

san7890 committed:
renaming pass through

fun fun fun

san7890 committed:
stuff

remember:
poster(?)
fix fridges(!)
billboards(!)
ensure power does workie(!)

yeah

san7890 committed:
Merge branch 'tgstation:master' into TheLizardsGas

san7890 committed:
we're almost there

just need posters and we should be good

san7890 committed:
finalization

let's go

san7890 committed:
Merge branch 'tgstation:master' into TheLizardsGas

san7890 committed:
Merge branch 'TheLizardsGas' of https://github.com/san7890/bruhstation into TheLizardsGas

SexmasterWasTaken commented:
More interesting space ruins is always cool, maybe give it a spawner for a gas station lizard employee

san7890 commented:
> More interesting space ruins is always cool, maybe give it a spawner for a gas station lizard employee

I really did think about this, but this is just such a small space with not a whole lot of content for that employee to do. It's meant to be abandoned, and the power for this station doesn't turn on until a player sets the SMES. I hope that enjoyable IC interactions can be formed after a player puts out the word or starts to establish this ruin, rather than have some poor player wait tens of minutes for nothing to happen.

san7890 committed:
minor fixes and patches

this was done over the course of a week so i missed some stuff on the first pass

Capsandi commented:
Ive always wanted a space gas station

san7890 committed:
i keep forgetting to save

largely huge skill issue

Imaginos16 commented:
You don't have the state of the art biodiesel they've been rolling out?

san7890 committed:
please pass CI now

rrrrrRRRRRRRRR

Bawhoppen reviewed (approved):
Good PR 👍

DragonTrance commented:
The layout dangerously reminds me of the gas the lizards ruin

san7890 committed:
GAS

a good (irl) friend of mine made the joke that they thought that "gas" referred to weed, so I thought that was pretty interesting/funny concept. I minangled some more lore in, some more spawners, etc.

san7890 committed:
FUCK

i did debug it! i just forgot to save! the skill issue you are looking for is in another castle mario

san7890 committed:
more lore

lore? LORE LORE LORE LOOOOORE. I'M GOING TO LOOOOOOOOOOOOOOOOOOOOOOOOORE

san7890 committed:
i'm so tired

YET I PERSIST.

san7890 commented:
I renamed it since it's more than just a mere change to a ruin in how it branches and adds several other bits of content to make it all work.

StoneMason commented:
the T is coming off the sign

DrDiasyl commented:
NOOOOOO, XENOPHOBIA IS NO MORE 😫😫

san7890 commented:
> the T is coming off the sign

This is intended.

optimumtact commented:
you need one of those signs that says "last gas station for next n miles" but the n is 100,000,000

san7890 commented:
> you need one of those signs that says "last gas station for next n miles" but the n is 100,000,000

Good idea.

Laz0rG commented:
removes soul, ligger appeasing 👎

san7890 committed:
roadsigns

i really like this idea.

san7890 committed:
Merge branch 'tgstation:master' into TheLizardsGas

san7890 commented:
![image](https://user-images.githubusercontent.com/34697715/151307027-97550bdd-1971-4ec9-b917-c811091ee93d.png)

optimumtact commented:
WHERE SIGN?
